deploy/helm-chart/templates/rbac.yaml

```diff
@@ -21,6 +21,7 @@ rules:
   - namespaces
   - services
   - secrets
+  - nodes
```

Comment:
Do we still need this now that we've removed the support for NodePort?

Reply:
Nope, good catch.